test/attr/attr_objc_typealias_inside_protocol.swift

```diff
@@ -0,0 +1,5 @@
+// RUN: %empty-directory(%t)
+// RUN: %swift-target-frontend -enable-objc-interop -parse-as-library %s -o /dev/null -disable-objc-attr-requires-foundation-module -module-name reduced -emit-module -emit-module-path %t/reduced.swiftmodule
```

Comment:
This should use `-c` to avoid trying to link.

Reply:
Done